src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -247,9 +310,17 @@ static int iadk_modules_get_configuration(struct processing_module *mod, uint32_
 					  uint32_t *data_offset_size, uint8_t *fragment,
 					  size_t fragment_size)
 {
+	if (mod->is_loadable) {
+		struct module_interface *mod_in =
+					(struct module_interface *)mod->priv.module_adapter;
+
+		return mod_in->get_configuration(mod, config_id, data_offset_size,
+						 fragment, fragment_size);
+	}
 	return iadk_wrapper_get_configuration(mod->priv.module_adapter, config_id,
-					      MODULE_CFG_FRAGMENT_SINGLE, *data_offset_size,
-					      fragment, fragment_size);
+					      MODULE_CFG_FRAGMENT_SINGLE,
+					      *data_offset_size, fragment,
+					      fragment_size);
```

Comment:
no change here, right? Better avoid unneeded unrelated changes, they only make reviews more difficult, we spend time trying to identify what you're changing here

Reply:
ok